### DISCUSSION TITLE
Is LSC Outdoor Floodlight (3200719) Supported?

### DISCUSSION THREAD

**Discussion by drschnalli** [ct-Open-Source/tuya-convert#1122] [General]
Hey there, i did not find my device in the compatibility list yet. Maybe its compatible i dont know.

Can you tell me if you successfully flashed the LSC Outdoor Floodlight (3200719 - LSC Smart Connect Außenbeleuchtung)

Thank you!

**Comment by Noschvie** [comment]
You will have to try it...
